Modules/Journey/QuestDetailsFrame.lua

```diff
@@ -28,22 +28,22 @@ function _QuestieJourney:DrawQuestDetailsFrame(container, quest)
 
     QuestieJourneyUtils:Spacer(container)
 
-    local questInfoHeader = _QuestieJourney:CreateHeading(QuestieLocale:GetUIString('JOURNEY_QUESTINFO'), true)
+    local questInfoHeader = _QuestieJourney:CreateHeading(l10n('Quest Information'), true)
     container:AddChild(questInfoHeader)
 
     -- Generic Quest Information
 
-    local levelLabel = _QuestieJourney:CreateLabel(Questie:Colorize(QuestieLocale:GetUIString('JOURNEY_QUEST_LEVEL'), 'yellow') .. quest.level, true)
+    local levelLabel = _QuestieJourney:CreateLabel(Questie:Colorize(l10n('Recommended Quest Level: '), 'yellow') .. quest.level, true)
```

Comment:
Remove the ` :` from the `l10n` part and hardcode it here. Same for the lines below

Reply:
Not a good idea since asian languages use `：` instead of `: `